ads_plugin.py

```diff
@@ -313,7 +313,6 @@ def on_load_png_button(self, event):
 
         # Check if the image format is valid
         image_extension = in_file.suffix
-        valid_extensions = [".png", ".tif", ".jpg", ".jpeg"]
```

Comment:
I'm unsure about this.

If the file is `.ome.tif`, it won't raise an error in the plugin itself.
It will however raise an error later in `imread` but the error message will only be displayed in the terminal and not in the plugin,

Should we apply the same strategies than in `imread` here? Both for fetching the extension and for the error message? or is the error in the terminal good enough?

Reply:
This sounds like a plugin issue (that it doesn't throw an error for ome.tif) and not an issue related to the specific aim of this PR, so I would say leave it the current way and open a new issue to handle the problem you raised about the plugin.